js/packages/candy-machine-ui/src/Home.tsx

```diff
@@ -215,14 +236,59 @@ const Home = (props: HomeProps) => {
     try {
       setIsUserMinting(true);
       document.getElementById('#identity')?.click();
+      let setupMint: SetupState | undefined;
       if (wallet.connected && candyMachine?.program && wallet.publicKey) {
+        if (neexTxnSplit && setupTxn === undefined) {
```

Comment:
Probably should rename the rest of the variables as well 😄 neexTxnSplit  -> needTxnSplit

Reply:
Whoops lol